addons/portal/static/src/chatter/frontend/chat_hub_service_patch.js

```diff
@@ -0,0 +1,8 @@
+import { chatHubService } from "@mail/core/common/chat_hub";
+import { patch } from "@web/core/utils/patch";
+
+patch(chatHubService, {
+    // When Chatter is loaded in portal environment chat hub is added to
+    // the chatter root component, see: portal_chatter.js
+    start() {},
```

Comment:
What is the point of this patch? This is basically an empty file

Reply:
It makes it so that the `chatHubService` doesn't load the `chathub` normal way inside the `mainComponentsContainer` as we would be loading the `chatHub` inside the `shadowDom` of `chatter`.

Reply:
I see we have another instance of this file in livechat. I guess it's fine then.